### ISSUE TITLE
IndexError Thrown when Export Calibrations with Configs & Constants after Delete 

### ISSUE THREAD

**Issue by joannekoch** [WHOIGit/ooicgsn-roundabout#313]
Steps to reproduce:

1. Create and Approve CCCs for a Part Template. (CCCs are Calibrations & Constants & Configs).
2. Create and Approve Constants, Calibration Coefficients and Configurations for Inventory associated with the Part.
3. Delete Calibrations for the Part Template.
4. Re add and Approve Calibrations for the Part Template.
5. Bulk Download Tool -> Export Calibrations with Configs & Constants.
6.  Get this IndexError Screen:

![indexerror](https://user-images.githubusercontent.com/65416034/105906710-0412d700-5ff2-11eb-9b3d-2002f73205d1.PNG)

Tested on v1.6.3

**Comment by sbatchelder** [comment]
Can I get the fill stacktrace for this? It'd be nice to know exactly on which line the error is being triggered on.

**Comment by sbatchelder** [comment]
(1) It's important to note that deleting a Calibration from the part template will delete/remove instances of that calibration on all inventory items that bear it. It's a destructive operation and also why the depricated checkbox is there for (I think). If you do delete and re-create a calibration on the part template, you'll have to re-create the calibration instance on your inventory item. 
(2) this is a legitimate bug that I've found will also occur if you try to export from an inventory item that has config/constants and SHOULD have calibrations, but doesn't. I've installed a fix and it will feature in my next pull request.
